scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -613,7 +613,9 @@ def process(self, data, handle_frame):
                     )
 
                     # skip invalid bytes which can appear because of the buffer overflow.
-                    frame_data = six.ensure_text(self._remaining, errors="ignore")
+                    frame_data = six.ensure_text(
+                        self._remaining, "utf-8", errors="ignore"
```

Comment:
``utf-8`` is already the default value for the ``encoding`` argument, but I explicitly added it here to make it more obvious and consistent with the rest of the code.